files/base.rb

```diff
@@ -63,6 +63,24 @@ def team_data(lookup_key = nil)
     data
   end
 
+  def check_data(lookup_key = nil)
```

Comment:
I just wanted helpers for check and client data.  I followed the existing pattern of the `team_data` method above.  I'm fine with a simplified version,  let me know what to do.